docker-compose.dev.yml

```diff
@@ -30,6 +31,29 @@ services:
     depends_on:
       - mongo
 
+  node:
```

Comment:
please, implement overriding of docker-compose.yml with docker-compose.dev.yml similar to how it is done in thenewboston-node